docs/tutorials/Containers.md

```diff
@@ -413,4 +419,31 @@ runtime {
 ```
 
 You can find the `sha256` of an image using `docker images --digests`
- 
\ No newline at end of file
+ 
+ 
+### Notes
+
+#### How does Cromwell know when a job or container has completed?
+Cromwell uses the presence of the `rc` (returncode) file to determine whether a task has succeeded or failed. This `rc` file is generated as part of the `script` within the execution directory, where the script is assembled at runtime. This is important as if the script executes successfully but the container doesn't terminate, Cromwell will continue the execution of the workflow and the container will persist hogging system resources.
+
+Within the two configurations above:
```

Comment:
You explain how these technologies run, but not how they relate to the generation of the rc file. Does `exec` running in the foreground mean that the container will exit properly?

Reply:
Yeah, I looked for specific wording in the documentation for this but couldn't find it. I tested it on the cluster and it runs in the foreground and terminates when complete, so have sort of asserted this. I think the point is important, and it would be great to have some more solid backing from the docs.